galasa-extensions-parent/buildSrc/src/main/groovy/galasa.extensions.gradle

```diff
@@ -4,7 +4,7 @@ plugins {
 }
 
 dependencies {
-    implementation 'dev.galasa:dev.galasa.framework:0.22.0'
+    implementation 'dev.galasa:dev.galasa.framework:0.27.0'
     implementation 'dev.galasa:dev.galasa:0.21.0'
```

Comment:
Should this be 0.27.0 ? Seems likely ?

Reply:
The latest version of dev.galasa is 0.21.0 https://mvnrepository.com/artifact/dev.galasa/dev.galasa

Not sure why it hasn't kept up with the releases.